js/kucoin.js

```diff
@@ -644,7 +648,7 @@ module.exports = class kucoin extends Exchange {
                         'max': this.safeNumber (market, 'baseMaxSize'),
                     },
                     'price': {
-                        'min': undefined,
+                        'min': priceIncerement,
```

Comment:
Are you sure this is right? `priceIncrement` usually differs from the min price. For instance, usually you can increment the BTC price by a cent (22999.12 -> 22999.13), but we can't set 0.01 USDT as the BTC price

Reply:
[kucoin api](https://docs.kucoin.com/#get-symbols-list) says so (you can search for `priceIncrement` keyword): 
> The priceIncrement field specifies the min order price as well as the price increment.This also applies to quote currency.
atm, in UI too, btc price can have only 1 digit after dot, and 0.1 seems to be the minimum inputable price there.

Reply:
@ttodua yeah 😅 
![image](https://user-images.githubusercontent.com/43336371/187507758-7c6e637b-e413-4a33-b771-7224954befdd.png)